serving/samples/rest-api-go/README.md

```diff
@@ -87,7 +87,7 @@ docker push "${REPO}/serving/samples/rest-api-go"
 
    - Use run this command:
 
-   ```
+   ```shell
```

Comment:
can we go without perl somehow?

Reply:
Yes. I avoided this because I liked the repo path in the YAML for ko apply since I don't have docker installed.

Updated to use envsubst like the traffic-split part of the example. Also simplified the image path so `ko publish -B` works well if you choose to use that over Docker.